postgres/assets/configuration/spec.yaml

```diff
@@ -336,6 +336,13 @@ files:
           value:
             type: number
             example: 10
+        - name: max_active_rows
```

Comment:
@djova do you have a better idea for how to name this config? It's not super clear what this is, but I can't think of something better 

maybe like `max_activity_batch_size`? ... IDK

Reply:
It's already under `query_activity` so we don't need to mention "activity" again. 

`payload_row_limit`? I'd include the word "payload" there to make it clearer that it's a payload limit, not a query limit on fetching from the DB.

Reply:
OK, `payload_row_limit` sounds fine